android/BOINC/app/src/main/java/edu/berkeley/boinc/client/Monitor.java

```diff
@@ -276,7 +280,9 @@ public void forceRefresh() {
     	if(Logging.DEBUG) Log.d(Logging.TAG,"forceRefresh()");
     	try{
     		updateTimer.schedule(new StatusUpdateTimerTask(), 0);
-    	} catch (Exception e){} // throws IllegalStateException if called after timer got cancelled, i.e. after manual shutdown
+    	} catch (Exception e){
+    		if(Logging.ERROR) Log.e(Logging.TAG,"Monitor.forceRefresh error: ",e);
```

Comment:
Not sure if `ERROR` is the right log level to use in this case. The comment seems to say that the exception is to be expected in certain conditions, doesn't it? If that's the case, shouldn't we just use `DEBUG`?

Reply:
Hm, thinking about it some more: yes, it might be expected in some conditions but you want to note it in case it's unexpected. So I'd say let's make this a `WARNING`.